.github/workflows/build-and-test.yml

```diff
@@ -104,7 +104,8 @@ jobs:
         run: yarn nx run ssr-e2e:e2e --headless
       # don't run build in parallel, can cause dead locks
       - name: Run Affected build
-        run: yarn nx affected:build
+      # @todo: @edbzn Nx cloud fail, disable it temporary.
+        run: NX_CLOUD_DISTRIBUTED_EXECUTION=false yarn nx affected:build
```

Comment:
why? :(

Reply:
and if we change the github actions, can we maybe disable `NX_VERBOSE_LOGGING` as well?